src/test/java/com/olgaboiar/mint/RequestTest.java

```diff
@@ -11,23 +11,23 @@
 
     @Test
     void getMethodReturnsGetMethodForGetRequest() throws MalformedURLException {
-        Request testRequest = new Request(new URL("http://0.0.0.0:5000/method_options"), "GET");
+        Request testRequest = new Request(new URL("http://0.0.0.0:5000/method_options"), "GET", "Content-Type: text/html");
         String actual = testRequest.getMethod();
 
         assertEquals("GET", actual);
     }
 
     @Test
     void getMethodReturnsPostMethodForPostRequest() throws MalformedURLException {
-        Request testRequest = new Request(new URL("http://0.0.0.0:5000/method_options"), "POST");
+        Request testRequest = new Request(new URL("http://0.0.0.0:5000/method_options"), "POST", "Content-Type: text/html");
         String actual = testRequest.getMethod();
 
         assertEquals("POST", actual);
     }
 
     @Test
     void getUriReturnsCorrectPath() throws MalformedURLException {
-        Request testRequest = new Request(new URL("http://0.0.0.0:5000/method_options"), "POST");
+        Request testRequest = new Request(new URL("http://0.0.0.0:5000/method_options"), "POST", "Content-Type: text/html");
         String actual = testRequest.getUri();
 
         assertEquals("/method_options", actual);
```

Comment:
It's a little bit misleading to require content-type for a request. If there's no request body (like a GET request), there's no content-type because there's not content. I think we want content-type to be optional as opposed to required.

Reply:
refactored in https://github.com/olgaboiar/MintHTTPServer/pull/14/commits/f69814852a141c7b701e3e7aeb70d83b893e2756